app/src/main/java/com/example/fitjournal/library/presentation/screen/library/utils/LibrarySearch.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.fitjournal.library.presentation.screen.library.utils
+
+import com.example.fitjournal.core.data.mockdata.MockData
+import com.example.fitjournal.library.presentation.screen.library.model.LibraryWorkoutItem
+import com.example.fitjournal.library.presentation.screen.library.model.WorkoutCategory
+import java.util.SortedMap
+
+fun searchForText(text: String): SortedMap<Char, List<String>> {
```

Comment:
Move the search for text to core under utils -> Filter package and add in a list as a param